packages/shared/src/consts/feature-tiers-constants.ts

```diff
@@ -462,12 +462,8 @@ function getConvertToMs(conversionToMs: boolean | undefined) {
   };
 }
 
-export function getFeatureForTierAsBoolean(
-  featureName: FeatureNameEnum,
-  tier: ApiServiceLevelEnum,
-  featureFlags: Partial<FeatureFlags>
-): boolean {
-  const feature: FeatureValue = getOriginalFeatureOrAugments(featureName, tier, featureFlags);
```

Comment:
for now, i was not able to find a reason for this logic at the current state without IS_2025_Q1_TIERING_ENABLED flag, if it will be needed, i will revert.